user/views.py

```diff
@@ -168,3 +168,56 @@ def delete_ride(request, ride_id):
     routesDB.delete_one({"_id": ride_id})
     return redirect("/myrides")
 
+
+def edit_user(request):
+    intializeDB() 
+    initializeCloud() 
+    user = userDB.find_one({"username": request.session['username']}) 
+
+    if request.method == 'POST':
+        form = EditUserForm(request.POST, request.FILES)  
+        if form.is_valid():
+            image = form.cleaned_data.get("profile_picture") 
+            if image: 
+                image.name = f"{form.cleaned_data['unityid']}.png" 
```

Comment:
Keep username as file name here instead of unity id. This will ensure it is overwritten on the cloud and we're not double storing images